calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
I think so. These menus from `Settings > Technical` will be moved under `Resources > Calendar` 
![cools](https://user-images.githubusercontent.com/11901801/29193977-7098b8aa-7ddc-11e7-9c4b-800dfe800290.JPG)

Perhaps only working time and resource leaves should go under `Resources > Calendar` and then that resources menu in the picture should go under `Resources > Resources`

Should I remove the existing menus from technical settings though?